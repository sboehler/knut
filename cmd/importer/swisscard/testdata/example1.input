Transaction date,Description,Card number,Currency,Amount,Debit/Credit,Status,Category
"01.01.1970","HANDLING FEE FOR PAPER STATEMENT","1234 56**** **7890","CHF","1.95","Debit","Posted",""
"02.02.1970","ANNUAL FEE","1234 56**** **7890","CHF","120.00","Debit","Posted",""
"03.03.1970","THRIFT SHOP","1234 56**** **7890","CHF","16.90","Debit","Posted","MISCELLANEOUS AND SPECIALTY RETAIL STORES"
"04.04.1970","GITHUB","1234 56**** **7890","CHF","-1.00","Credit","Posted","COMPUTER SOFTWARE STORES"
"05.05.1970","SBB","1234 56**** **7890","CHF","18.00","Debit","Posted","PASSENGER RAILWAYS"
"06.06.1970","GITHUB","1234 56**** **7890","CHF","1.05","Debit","Posted","COMPUTER SOFTWARE STORES"
