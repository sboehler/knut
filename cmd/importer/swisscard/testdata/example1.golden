1970-01-01 "HANDLING FEE FOR PAPER STATEMENT"
Liabilities:CreditCard Expenses:TBD                 1.95 CHF

1970-02-02 "ANNUAL FEE"
Liabilities:CreditCard Expenses:TBD                  120 CHF

1970-03-03 "THRIFT SHOP / MISCELLANEOUS AND SPECIALTY RETAIL STORES"
Liabilities:CreditCard Expenses:TBD                 16.9 CHF

1970-04-04 "GITHUB / COMPUTER SOFTWARE STORES"
Expenses:TBD           Liabilities:CreditCard          1 CHF

1970-05-05 "SBB / PASSENGER RAILWAYS"
Liabilities:CreditCard Expenses:TBD                   18 CHF

1970-06-06 "GITHUB / COMPUTER SOFTWARE STORES"
Liabilities:CreditCard Expenses:TBD                 1.05 CHF

